### PR TITLE
Task8c - added addOrder method into OrderModel; added dummy data into Controller

### PR DESCRIPTION
**Pull request by amandahrdmn**

### CODE REVIEW

File: src/Controllers/AddOrderController.php [hunk 1]
```diff
@@ -21,6 +22,35 @@ public function __construct($orderModel)
 
     public function __invoke(Request $request, Response $response, array $args)
     {
+
```
Comment (MaxGranger97) [inline src/Controllers/AddOrderController.php:25]: Whitespace?

File: src/Models/OrderModel.php [hunk 2]
```diff
@@ -16,4 +17,64 @@ public function __construct($db)
     {
         $this->db = $db;
     }
+
+    public function addOrder(OrderEntityInterface $orderEntity)
```
Comment (turnerror) [inline src/Models/OrderModel.php:21]: definitely add a doc comment for this haha

File: src/Models/OrderModel.php [hunk 3]
```diff
@@ -16,4 +17,64 @@ public function __construct($db)
...
+                                                :shippingAddress1,
+                                                :shippingAddress2,
+                                                :shippingCity,
+                                                :shippingPostcode,
+                                                :shippingCountry)");
+
+        $orderQueryResult = $orderQuery->execute($order);
+
+        foreach($orderedProducts as $product) {
+            $productList[] = [$product['newStockLevel'], $product['sku']];
```
Comment (turnerror) [inline src/Models/OrderModel.php:56]: I think this isn't needed?

File: src/Models/OrderModel.php [hunk 4]
```diff
@@ -16,4 +17,80 @@ public function __construct($db)
...
+
+
+    /**
+     * Adds an order to the Database, including adding order in in orders table,
+     * adding rows in linking table for order/products relationships,
+     * and updates products' stockLevels based on calculated newStockLevels
+     * @param OrderEntityInterface $orderEntity
+     * @return bool
+     * true: if all three db queries have been added successfully to DB
+     * false: if any of the three db queries fail
```
Comment (turnerror) [inline src/Models/OrderModel.php:29]:
```suggestion
     * Adds an order to the Database, including inserting order in orders table,
     * adding products ordered in linking table for order/products relationships,
     * and updates products' stockLevels with newStockLevels after order volume is taken into account.
     * @param OrderEntityInterface $orderEntity
     * @return bool depending on whether the transaction was successful or not.
```

File: src/Models/OrderModel.php [hunk 5]
```diff
@@ -16,4 +17,77 @@ public function __construct($db)
     {
         $this->db = $db;
     }
+
+    /**
+     * Adds an order to the Database, including inserting order in orders table,
+     * adding products ordered in linking table for order/products relationships,
+     * and updates products' stockLevels with newStockLevels after order volume is taken into account.
```
Comment (turnerror) [inline src/Models/OrderModel.php:24]:
```suggestion
     * Adds an order to the Database which does the following in a transaction: 
     *  - adds order into the orders table
     *  - adds products ordered into the productsOrdered linking table
     *  - updates products' stockLevels with newStockLevels after order volume is taken into account.
```

**Pull request merged**